src/containers/PaymentInfo.tsx

```diff
@@ -8,20 +8,28 @@ import {
 } from 'react-native';
 import { Button } from 'react-native-elements';
 import { RootStackParamList } from '../../App';
+import { RouteProp } from '@react-navigation/native';
 import { StackNavigationProp } from '@react-navigation/stack';
 import Form from './auth/Form';
+import ItemCatalogueService from '../services/ItemCatalogueService';
 import React, { useState } from 'react';
 
 import Loading from '../components/Loading';
 
 interface PaymentInfoProps {
 	navigation: StackNavigationProp<RootStackParamList, 'PaymentInfo'>;
+	route: RouteProp<RootStackParamList, 'PaymentInfo'>;
 }
 
 const screenWidth = Dimensions.get('window').width;
 const screenHeight = Dimensions.get('window').height;
 
-const PaymentInfo = ({ navigation }: PaymentInfoProps) => {
+const PaymentInfo = ({ navigation, route }: PaymentInfoProps) => {
+	/* OG Bot is BruinBear with id 5ff798be0390ab19822d21db for demo purposes (from local deb db),
+			but if you come from QR view, botId will be set from navigation params */
+	const [botId] = useState<string>(
```

Comment:
I don't think `useState` is necessary here

```suggestion
	const botId = 
```